dask/dataframe/io/tests/test_csv.py

```diff
@@ -784,6 +785,19 @@ def test_auto_blocksize():
     assert auto_blocksize(5000, 2) == 250
 
 
+def test__infer_block_size(monkeypatch):
```

Comment:
Sorry, I should have included this in my initial review. Since the purpose of this test may not be obvious, could you add a small comment providing a bit of context. Perhaps something like:

```
# psutil returns a total memory of `None` on some systems
# see https://github.com/dask/dask/pull/7601
```

Reply:
Sure! I just added this.